website/docs/topics/feature-flags/feature-flag-best-practices.mdx

```diff
@@ -191,4 +229,20 @@ To ensure a good developer experience, you should provide the following:
 - **Usage metrics**: Provide aggregated insights into feature flag usage, helping developers confirm that everything is working as expected.
 - **Documentation and training**: Offer clear, comprehensive documentation for the API, UI, and SDKs, with easy-to-follow examples. This simplifies onboarding for new developers and supports continuous training, ensuring the effective use of the feature flagging system.
 
-Thank you for reading. Our motivation for writing these principles is to share what we've learned building a large-scale feature flag solution with other architects and engineers solving similar challenges. Unleash is open-source, and so are these principles. Have something to contribute? [Open a PR](https://github.com/Unleash/unleash/pulls) or [discussion](https://github.com/orgs/Unleash/discussions) on our GitHub.
+---
+
+## Additional resources
+
+Thank you for reading. Our motivation for writing these principles is to share what we've learned building a large-scale feature flag solution with other architects and engineers solving similar challenges. Unleash is open-source, and so are these principles.
+Have something to contribute? [Open a PR](https://github.com/Unleash/unleash/pulls) or [discussion](https://github.com/orgs/Unleash/discussions) on our GitHub.
+
```

Comment:
I would get rid of everything after this line

I'm not sure we want to give people even more information that they haven't asked for, but we could put a link to our pricing page if they're curious

Reply:
It was a suggestion from Ercule to add this section on additional resources: https://docs.google.com/document/d/1aCf0leHvrB9sVjeG3N0icPcUo7y5FcgZz03qX5JvL1Y/edit?tab=t.0

Given it's a page that drives a lot of traffic, I think it makes sense to give people some ideas of where to go next to learn more. The pricing page feels a bit too direct